test/run_envoy_tests.sh

```diff
@@ -61,10 +61,16 @@ then
   exit 0
 fi
 
+# TODO(htuch): Clean this up when Bazelifying the hot restart test below.
+HOT_RESTART_JSON="$TEST_SRCDIR"/test/config/integration/hot_restart.json
+cat "$TEST_TMPDIR"/test/config/integration/server.json |
```

Comment:
Added #654. I like the idea of making this an admin endpoint feature, but if we bind the admin http address to 0 as well as we'd like to in tests, we need to find the admin endpoint. Maybe we could just have a flag that provides this address and return the rest from the admin endpoint?